sdk/core/Azure.Core.TestFramework/src/TestProxy.cs

```diff
@@ -81,6 +81,13 @@ private TestProxy(string proxyPath, bool debugMode = false)
                 }
             };
 
+            bool.TryParse(Environment.GetEnvironmentVariable("TF_BUILD"), out bool inCI);
```

Comment:
![image](https://github.com/Azure/azure-sdk-for-net/assets/45376673/5d577eba-4a92-46dd-a88d-d79e5114db8a)

Reply:
Most of our build-related scripts use `$ci` as a flag.   Maybe we should consider naming this `CI_BUILD` for consistency?

Reply:
This env variable is defined by the CI host environment, so I don't think we have a choice on naming.